.github/workflows/ci.yml

```
@@ -1027,7 +1025,7 @@ jobs:
         uses: ./.github/actions/migration_tests
       - name: >
           Tests: ${{matrix.python-version}}:${{needs.build-info.outputs.parallel-test-types-list-as-string}}
-        run: breeze testing tests --run-in-parallel
+        run: breeze testing tests --run-in-parallel --only-run-db-tests
```

nit: I like having the verb first (only my opinion :))

```suggestion
        run: breeze testing tests --run-in-parallel --run-db-tests-only
```

---

Agree. It was inconsistent as well. Will change.